java/tools/src/java/org/apache/orc/tools/Driver.java

```diff
@@ -116,6 +116,8 @@ public static void main(String[] args) throws Exception {
       ConvertTool.main(conf, options.commandArgs);
     } else if ("key".equals(options.command)) {
       KeyTool.main(conf, options.commandArgs);
+    } else if ("count".equals(options.command)) {
```

Comment:
Could you add a description for `count` command after line 95?
- https://github.com/apache/orc/pull/506/files#diff-e0682227343a18a7d50605f939f02795R95